### PR TITLE
Matthew/swa readme

### PR DESCRIPTION
**Pull request by growlix**
Updated SWA method card to current format and added readme. Typo fix to gradient clipping readme.

### CODE REVIEW

File: composer/algorithms/swa/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+|:--|
+|*Visualization of SWA from an extensive [PyTorch blogpost about SWA](https://pytorch.org/blog/stochastic-weight-averaging-in-pytorch/).*|
+
+
+## How to Use
+
+### Composer Trainer
+
+<!-- TODO: Address timeouts -->
+<!--pytest-codeblocks:skip-->
```
Comment (mvpatel2000) [inline composer/algorithms/swa/README.md:19]: Can you enable this test? See https://github.com/mosaicml/composer/pull/1279 which should be merged any sec for a few examples

Comment (growlix) [reply composer/algorithms/swa/README.md]: Lmk if this looks right

File: composer/algorithms/swa/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+swa_algorithm = SWA(
+    swa_start="1ep",
+    swa_end="2ep",
+    update_interval='5ba',
+)
+trainer = Trainer(
+    model=model,
+    train_dataloader=train_dataloader,
+    eval_dataloader=eval_dataloader,
+    max_duration="3ep",
```
Comment (mvpatel2000) [inline composer/algorithms/swa/README.md:46]: Nit: can we make 2ep? Slightly faster and consistent with other tests

File: composer/algorithms/swa/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    algorithms=[swa_algorithm]
+)
+
+trainer.fit()
+
```
Comment (mvpatel2000) [inline composer/algorithms/swa/README.md:51]: Nit: extra line after `trainer.fit()`

**Review by mvpatel2000 — APPROVED** [review]
LGTM! The original PR has one small thing I want to run by Ravi before merging, so this will have to be held off for a bit, but should be good to go once thats one.

File: composer/algorithms/swa/README.md (hunk 2 above)
Comment (growlix) [reply composer/algorithms/swa/README.md:46]: Due to the nature of the algorithm, it doesn't make sense to run it for less than 3 epochs. I know this is an example, but I think the example should reflect the best practices of the method. Can go into detail if you're curious, but happy to change if necessary.
